backends/gdx-backend-headless/src/com/badlogic/gdx/backends/headless/HeadlessApplication.java

```diff
@@ -70,6 +70,7 @@ public HeadlessApplication (ApplicationListener listener, HeadlessApplicationCon
 		// they are only implemented as mock objects
 		this.graphics = new MockGraphics();
 		this.graphics.setForegroundFPS(config.updatesPerSecond);
+		this.graphics.setBackgroundFPS(config.updatesPerSecond);
```

Comment:
Headless apps don't have a concept of foreground or background, so I don't see any point in adding setBackgroundFPS() to anything headless...